pxe/PXELinux_default.erb

```diff
@@ -17,6 +17,13 @@ LABEL local
      MENU DEFAULT
      LOCALBOOT 0
 
+LABEL discovery
+     MENU LABEL (discovery)
+     MENU DEFAULT
```

Comment:
The grey bar is at ```(discovery)``` in my setup then, although ```(local)``` gets booted. That may depend on the PXELinux version, but to be sure, I would delete this line.